commit fb2bbd4b64ab20e11475a163a9dcaefb86802ad7
Author: Stephen Segal <ssegal@meraki.com>
Date:   Sun Oct 16 20:25:10 2016 -0700

    fix namespace collison
    
    posix_close will be the name of a new function in the POSIX standard.
    musl already supports this, resulting in a collision.

diff --git a/src/slposio.c b/src/slposio.c
index 9ce9697..fb675e8 100644
--- a/src/slposio.c
+++ b/src/slposio.c
@@ -363,7 +363,7 @@ static int posix_close_fd (int *fd)
    return 0;
 }
 
-static int posix_close (SLFile_FD_Type *f)
+static int posix_close_slfile (SLFile_FD_Type *f)
 {
    int status = do_close (f);
 
@@ -1001,7 +1001,7 @@ static SLang_Intrin_Fun_Type Fd_Name_Table [] =
    MAKE_INTRINSIC_2("write", posix_write, V, F, B),
    MAKE_INTRINSIC_1("dup_fd", posix_dup, V, F),
    MAKE_INTRINSIC_2("dup2_fd", posix_dup2, I, F, I),
-   MAKE_INTRINSIC_1("close", posix_close, I, F),
+   MAKE_INTRINSIC_1("close", posix_close_slfile, I, F),
    MAKE_INTRINSIC_1("_close", posix_close_fd, I, I),
 #if defined(TTYNAME_R)
    MAKE_INTRINSIC_0("ttyname", posix_ttyname, V),
